tools/run_tests/performance/loadtest_config.py

```diff
@@ -383,7 +383,17 @@ def main() -> None:
     if args.runs_per_test < 1:
         argp.error('runs_per_test must be greater than zero.')
 
-    substitutions = parse_key_value_args(args.substitutions)
+    # Config generation ignores environment variables that are passed by the
+    # controller at runtime.
+    substitutions = {
```

Comment:
Nit: It looks like there's a coupling here between this list and the environment variables specified by the driver. That is, when someone adds a new environment variable to the controller, they also have to know to add that environment variable here. This seems like a difficult problem to solve considering that the controller lives in a separate repo, but if you can think of a simple solution, it might be good to use it.

Reply:
Disregard this. https://github.com/grpc/grpc/pull/26516 made clear that there is no inter-repo coupling here.